docs/integration/chip/nxp.rst

```diff
@@ -50,30 +50,31 @@ Basic initialization:
 PXP draw initialization is done automatically in :cpp:func:`lv_init()` once the
 PXP is enabled, no user code is required:
 
-.. code:: c
+.. code-block:: c
 
-  #if LV_USE_DRAW_PXP
-    lv_draw_pxp_init();
-  #endif
+    #if LV_USE_DRAW_PXP
+        lv_draw_pxp_init();
+    #endif
 
 During PXP initialization, a new draw unit `lv_draw_pxp_unit_t` will be created
 with the additional callbacks:
 
-.. code:: c
+.. code-block:: c
 
     lv_draw_pxp_unit_t * draw_pxp_unit = lv_draw_create_unit(sizeof(lv_draw_pxp_unit_t));
     draw_pxp_unit->base_unit.evaluate_cb = _pxp_evaluate;
     draw_pxp_unit->base_unit.dispatch_cb = _pxp_dispatch;
     draw_pxp_unit->base_unit.delete_cb = _pxp_delete;
 
+
 and an addition thread `_pxp_render_thread_cb()` will be spawned in order to
 handle the supported draw tasks.
 
-.. code:: c
+.. code-block:: c
 
-  #if LV_USE_PXP_DRAW_THREAD
-    lv_thread_init(&draw_pxp_unit->thread, LV_THREAD_PRIO_HIGH, _pxp_render_thread_cb, 2 * 1024, draw_pxp_unit);
-  #endif
+    #if LV_USE_OS
+        lv_thread_init(&draw_pxp_unit->thread, LV_THREAD_PRIO_HIGH, _pxp_render_thread_cb, 2 * 1024, draw_pxp_unit);
+    #endif
```

Comment:
How come this was changed? https://github.com/lvgl/lvgl/blob/9db0ee3d02a113058c8296d68eee98baa7967d6f/src/draw/nxp/pxp/lv_draw_pxp.c#L94-L96

Reply:
@liamHowatt Oh my, my oversight.  That was outdated code from a cherry-pick that I overlooked.  It was only meant to include the indentation change for consistency.  Fixed in current commit.